mvrx/build.gradle

```diff
@@ -2,19 +2,20 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
-apply plugin: "jacoco"
-apply plugin: "com.vanniktech.maven.publish"
+apply plugin: 'jacoco'
+apply plugin: 'com.vanniktech.maven.publish'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
     resourcePrefix 'mvrx_'
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 28
+        minSdkVersion versions.minSdk
+        targetSdkVersion versions.targetSdk
         consumerProguardFiles 'proguard-rules.pro'
     }
-    buildToolsVersion "$buildToolsVersion"
+
```

Comment:
do we need
```
  kotlinOptions {
    jvmTarget = "1.8"
  }
```

in here too, since we are targeting 1.8 in compileOptions?

Reply:
As far as I could understand from [Google Android documentation](https://developer.android.com/studio/write/java8-support), this is required bye Kotlin compiler:
```Gradle
android {

  // Configure only for each module that uses Java 8
  // language features (either in its source code or
  // through dependencies).

  compileOptions {
    sourceCompatibility JavaVersion.VERSION_1_8
    targetCompatibility JavaVersion.VERSION_1_8
  }

  // For Kotlin projects
  kotlinOptions {
    jvmTarget = '1.8'
  }

}
```

Reply:
@elihart Found it! [Cannot inline bytecode built with Java 8 into code built with targed Java 6](https://stackoverflow.com/questions/48988778/cannot-inline-bytecode-built-with-jvm-target-1-8-into-bytecode-that-is-being-bui/56996020#56996020)